eos-prune-printers.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Endless prune printers
+After=cups.service
```

Comment:
Since this depends on CUPS running (or `lpstat` and `lpadmin` will fail), I think adding `Requires=cups.service` is a good idea.

Reply:
yap